.azure/pipelines/ci.yml

```diff
@@ -431,6 +431,10 @@ jobs:
     - bash: "./eng/scripts/install-nginx-mac.sh"
       displayName: Installing Nginx
     afterBuild:
+    - bash: ./build.sh --ci --pack --no-build --no-restore --no-build-deps "/bl:artifacts/log/packages.pack.binlog"
```

Comment:
I don't like the fact that we need to run these steps separately, since if we were to move these tests to another configuration, we need to move these steps as well. However I'd rather unblock these tests first and investigate simplifying via a script or MSBuild magic separately.